aldryn_config.py

```diff
@@ -2,12 +2,10 @@
 
 
 class Form(forms.BaseForm):
+
     def to_settings(self, data, settings):
-        try:
```

Comment:
This is crucial for versioning-filer to overwrite filer code, this order has to be kept for the addons. This code needs to be added back.

Reply:
Fixed